### DISCUSSION TITLE
Parking plugin: is it still alive? + translation improvment

### DISCUSSION THREAD

**Discussion by ldmpub** [osmandapp/OsmAnd#15497] [Q&A]
Hi,

I want to improve the french translation of the Parking plugin.
Can someone point me out to its translation? (I can not find it in Weblate where I'm used to translate Osmand App)

Note: this plugin seems to be very old, don' t hesitate to tell me if performing any enhancement is useless ...

Thanks for your answer,

**Comment by vshcherb** [comment]
Parking plugin is integrated in the app itself and it's not a separate app-plugin any more

**Reply by ldmpub** [reply]
Thanks for the answer. This "all in one" approach don't ease translation.

BTW is it possible to make this addon evolve? Or is it "frozen" in terms of dev?

**Reply by vshcherb** [reply]
Separate application is frozen.
